### DISCUSSION TITLE
[Feature] Store Star rating (and others) in image meta data instead of sidecar XMP

### DISCUSSION THREAD

**Discussion by r00tr4v3n** [immich-app/immich#11794] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Store metadata in the image file itself instead of a sidecar XMP file. I noticed this in v1.112
Some people might prefer not to double the amount of files they have.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
As a design principle, we don't modify the original file

**Reply by r00tr4v3n** [reply]
I see. So there is no chance of going the Digikam way of choice? [https://www.digikam.org/news/2011-05-12_new_features_in_digikam_2.0_xmp_sidecar/](url)

**Reply by Bierchermuesli** [reply]
hey @r00tr4v3n how did you proceed with this? I understeand the immich design principle but on the long run I would like to store these information in the file to keep things together.... any ideas for this? e.g. (just stupid) periodically download the xmp files to my local db and merge them with digikam?
